ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -615,36 +613,27 @@ object Config {
             "Enable the development version of the Daml-LF language. Highly unstable. Should not be used in production."
           )
 
-        // TODO append-only: remove after removing support for the current (mutating) schema
+        // TODO append-only: remove
         opt[Unit]("index-append-only-schema")
           .optional()
           .text("Legacy flag with no effect")
           .action((_, config) => config)
 
+        // TODO remove
         opt[Unit]("mutable-contract-state-cache")
```

Comment:
Why not remove this at the same time?

Reply:
to prevent breakage in LR and VMBC just created a tracker story do environment maintenance and final cleanup of configurations (for example we have sort of the same with append-only still)

Reply:
I think it's OK to just fix any issues as we upgrade, but your call.